bin/attach-dev

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env bash
+
+# DOC: Connect to the app running locally using Docker. Alias of 'docker attach civiform'.
+
+source bin/lib.sh
+
+echo "Attaching to civiform.  Run Ctrl-P, followed by Ctrl-Q, to detach"
+echo "Ctrl-C will quit sbt, re-attach to restart"
+
+# Start a warmup request loop so that when the subsequent server startup is ready, we trigger lazy
+# loading its edit-refresh resources so save developer time.
+for i in {1..60}; do
+  sleep 5
+  curl --output /dev/null --silent --fail http://localhost:9000/loginForm && break
```

Comment:
question (blocking): i think this might stop the execution of the script if it fails since `set -e`, this may need to be
```suggestion
  if curl --output /dev/null --silent --fail http://localhost:9000/loginForm; then
    break
  fi
```

Reply:
It doesn't stop the script (since this runs in the background), but it might very well be stopping the loop.

Changed